wvm-apps/wvm-exexed/crates/reth-exexed/src/exex/ar_process.rs

```diff
@@ -0,0 +1,128 @@
+use crate::network_tag::get_network_tag;
+use crate::new_etl_exex_biguery_client;
+use crate::util::check_block_existence;
+use arweave_upload::{ArweaveRequest, UploaderProvider};
+use exex_wvm_bigquery::repository::StateRepository;
+use exex_wvm_da::{DefaultWvmDataSettler, WvmDataSettler};
+use reth::primitives::revm_primitives::alloy_primitives::BlockNumber;
+use reth_primitives::SealedBlockWithSenders;
+use std::sync::Arc;
+use tokio::sync::mpsc::Sender;
+use tokio::task::JoinHandle;
+use tracing::{error, info};
+use wvm_borsh::block::BorshSealedBlockWithSenders;
+use wvm_static::SUPERVISOR_RT;
+
+pub struct ArProcess {
+    buffer_size: usize,
+    sender: Sender<(SealedBlockWithSenders, String)>,
+    thread: JoinHandle<()>,
+}
+
+impl ArProcess {
+    pub fn new(buffer_size: usize) -> Self {
+        let (sender, mut receiver) = tokio::sync::mpsc::channel(buffer_size);
+
+        let thread = SUPERVISOR_RT.spawn(async move {
+            let irys_provider = UploaderProvider::new(None);
+            let big_query_client = new_etl_exex_biguery_client().await;
+            let state_repo = StateRepository::new(Arc::new(big_query_client));
+
+            while let Some(msg) = receiver.recv().await {
+                tokio::spawn(async {
+                    let sealed_block: SealedBlockWithSenders = msg.0;
+                    let notification_type: &str = msg.1.as_str();
+
+                    let block_number = sealed_block.block.header.header().number;
+                    let block_hash = sealed_block.block.hash().to_string();
+
+                    let sealed_block_clone = sealed_block.clone();
+
+                    let brotli_borsh = match Self::serialize_block(sealed_block_clone) {
```

Comment:
"borsh_brotli" var name instead of "brolti_borsh" to follow the logic and naming convention of the Arweave tags as well ("Encoding")